ui.tests/test-module/specs/wizard/wizard.runtime.repeatability.cy.js

```diff
@@ -151,6 +151,43 @@ describe("Form with Wizard Container", () => {
             getWizardPanels().should('have.length', 6);
         })
     });
+
+    it("After clicking on every tab, if tab-1 is repeated, the repeated instance should not have stepped class", () => {
+        getTabs().should('have.length', 4);
+        getWizardPanels().should('have.length', 4);
```

Comment:
I don't see code to add/remove instances in between the panel, which was asked as per last review comments

Reply:
I have added the instances in tabs/wizards, you can check it in the test case as well.